app/views/just_so_you_know/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for(:template_name, "card") %>
+
+<%= form_with url: just_so_you_know_steps_path, method: :put, local: true do %>
+  <%= fields_for(:form, @form, builder: Cfa::Styleguide::CfaFormBuilder) do |f| %>
+    <div class="form-card__header spacing-below-35">
+      <div class="emoji emoji--bigger spacing-below-25 spacing-below-25 emoji--credit-card"></div>
+      <h2 class="form-card__title">
+        Just so you know.
+      </h2>
+    </div>
+    <div class="form-card__content">
+      <div class="grid">
+        <div class="grid__item">
+          <p class="with-padding-small text--left">
+            The card will have the oldest student&apos;s name on it when it arrives. Adults in the household will be able to use it to shop for the whole family.

```

Comment:
This is missing the second sentence of the copy from Figma

Reply:
Looks like the copy for this was changed over the weekend ... and not just the additional sentence. New copy added. Also, the copy from Figma is slightly incorrect: it references the `oldest students' name` where it should be the `oldest student's` name. Student, in this case, is specifically singular (there can be only one oldest student), and therefore, the singular form of the possessive is desired here.

Changes added to PR.